src/screens/Settings/index.css

```diff
@@ -151,14 +151,16 @@
 .warning {
   color: yellow;
   font-size: 12px;
-  width: 513px;
+  width: 100%;
+  max-width: 513px;
   white-space: break-spaces;
   padding-top: 4px;
 }
 
 .smallLink {
   font-size: 12px;
-  width: 513px;
+  width: 100%;
+  max-width: 513px;
```

Comment:
all the changes in this file are needed to fix some responsiveness issues with high zoom levels